dls-controls/malcolmjs
# Feature/refactor url handling #206

## Description

Updated all the url handling so it now works with a tree of objects. Various updates throughout the code to get the nav elements to populate.

## Testing instructions

Add a set up instructions describing how the reviewer should test the code
- Review code
- Check Travis build
- Review changes to test coverage
- Run the website and navigate to `/gui/PANDA/layout/SEQ2/table/.info`

## Agile board tracking

connect to #206 
closes #206
